docker/development/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # Apache License 2.0
 
-FROM ubuntu:20.04
+FROM ubuntu:20.04 as root-user
```

Comment:
Is this necessary?

Reply:
i use the same template for build all the image and is hard coded on it https://github.com/open-space-collective/open-space-toolkit/blob/main/.github/workflows/build-image.yml

Reply:
Yes it is because he the github workflow file in each repo uses the `root-user` as the "target", and also its consistent with the other dockerfiles for all the other ostk-astro repos